packages/graphql/tests/integration/types/duration.int.test.ts

```diff
@@ -52,12 +51,10 @@ describe("Duration", () => {
             });
 
             const id = generate({ readable: false });
-            const years = faker.random.number({ min: -6, max: 6 });
-            const months = faker.random.number({ min: -10, max: 10 });
-            const days = faker.random.number({ min: -50, max: 50 });
-            const minutes = faker.random.float();
+            const hours = 2;
+            const minutes = 30;
 
-            const duration = `P${years}Y${months}M${days}DT${minutes}M`;
+            const duration = `PT${hours}H${minutes}M`;
             const parsedDuration = parseDuration(duration);
```

Comment:
This will be somewhat difficult to test in that case because the driver doesn't allow passing a string as a duration. Since it only takes days months seconds and nanoseconds, the string stored in the database that represents the duration is not necessarily the same as the one being defined in the test. I would have to recreate that function in this test otherwise.

This is why there is an explicit `Duration.test.ts` file that tests various strings to make sure `parseDuration` is working correctly. It just happened that when using floating point numbers from `faker` it was randomly exposing an edge case not covered by that test.

Reply:
Fair enough, this proposal was outside the scope of the PR anyway, so happy to approve as is